src/client/Client.cc

```diff
@@ -4140,10 +4140,11 @@ int Client::mark_caps_flushing(Inode *in, ceph_tid_t* ptid)
   in->flushing_cap_tids[flush_tid] = flushing;
 
   if (!in->flushing_caps) {
-    ldout(cct, 10) << "mark_caps_flushing " << ccap_string(flushing) << " " << *in << dendl;
+    ldout(cct, 10) << __func__ << " newly flush " << ccap_string(flushing) << " " << *in << dendl;
     num_flushing_caps++;
   } else {
-    ldout(cct, 10) << "mark_caps_flushing (more) " << ccap_string(flushing) << " " << *in << dendl;
+    ldout(cct, 10) << __func__ << " newly flush " << ccap_string(flushing) 
+                   << " currently is flushing " << ccap_string(in->flushing_caps) << " " << *in << dendl;
```

Comment:
'<< *in ' already includes flushing_caps

Reply:
thank you for your review
forgot to check "<< *in", 
i will rebase it